lib/main.dart

```diff
@@ -37,8 +36,8 @@ class HomePage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    context.read<AuthBloc>().add(const AuthEventInitialize());
-    return BlocConsumer<AuthBloc, AuthState>(
+    context.read<AuthCubit>().initialize();
+    return BlocConsumer<AuthCubit, AuthState>(
```

Comment:
we can optimize consumer or listener with `listenWhen`, will reduce the number of listen